src/main/java/org/truffleruby/language/SafepointManager.java

```diff
@@ -270,4 +292,18 @@ private void interruptOtherThreads() {
         }
     }
 
+    public void shutdown() {
+        if (!runningThreads.isEmpty()) {
+            Log.LOGGER.warning("still threads registered with safepoint manager at shutdown:\n" + context.getThreadManager().getThreadDebugInfo() + getSafepointDebugInfo());
```

Comment:
What are `still threads`?
I think `Threads are still registered with the safepoint manager at shutdown:` is easier to read.

Reply:
#484